tests/phpunit/tests/rest-api/class-wp-rest-block-directory-controller-test.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * Test WP_REST_Block_Directory_Controller_Test()
+ *
+ * @package Gutenberg
+ * phpcs:disable
+ */
+class WP_REST_Block_Directory_Controller_Test extends WP_UnitTestCase {
+	protected $controller = null;
+
+	function setUp() {
+		parent::setUp();
+
+		global $wp_rest_server;
+		$wp_rest_server = new \WP_REST_Server;
+		$this->server   = $wp_rest_server;
+		do_action( 'rest_api_init' );
+
+		$this->controller = new WP_REST_Block_Directory_Controller();
+		$this->controller->register_routes();
```

Comment:
This shouldn't be necessary, adding the registration in `create_initial_rest_routes` should be sufficient for the tests.